components/usage-api/usage/v1/usage.proto

```diff
@@ -4,16 +4,35 @@ package usage.v1;
 
 option go_package = "github.com/gitpod-io/gitpod/usage-api/v1";
 
-import "usage/v1/pagination.proto";
+import "google/protobuf/timestamp.proto";
 
 service UsageService {
-    // GetUsage retrieves all usage for a team.
-    rpc GetUsage(GetUsageRequest) returns (GetUsageResponse) {}
+    // GetBilledUsage retrieves all usage for the specified attributionId
+    rpc GetBilledUsage(GetBilledUsageRequest) returns (GetBilledUsageResponse) {}
 }
 
-message GetUsageRequest {
-    Pagination pagination = 1;
+message GetBilledUsageRequest {
+    string attribution_id = 1;
```

Comment:
I guess it doesn't matter since we never used it, but the field number here normally shouldn't re-use something that was previously assigned to something else.

Reply:
True, but we have to be conscious about what we carry around with us, because it tends to create veeeery long-living drag.
Removing a never-used attribute is a good choice I think. :+1: